### PR TITLE
GameDB: Even more GS HW Fixes

### PR DESCRIPTION
**Pull request by RedDevilus**
### Description of Changes

Affects:
- Ape Escape 3
- Armored Core Last Raven
- Berserk
- Forever Kingdom
- Front Mission 4 / 5
- Need For Speed Underground 1 / 2
- Kings Field 4 - The Ancient City
- Test Drive Unlimited
- Jak 2 / 3 / X
### Rationale behind Changes

Less tinkering.
### Suggested Testing Steps

Test the relevant games.

### CODE REVIEW

File: bin/resources/GameIndex.yaml [hunk 1]
```diff
@@ -2509,6 +2531,8 @@ SCES-50781:
   name: "Destruction Derby Arena [Beta, Promo, & Full Retail]"
   region: "PAL-M6"
   compat: 5
+  gsHWFixes:
+    disableDepthSupport: 1 # Removes fog-like filter.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:2535]: Don't add these for DDS, this is a bug in the TC which is being fixed in #5349

File: bin/resources/GameIndex.yaml [hunk 2]
```diff
@@ -2883,6 +2908,8 @@ SCES-51977:
 SCES-51979:
   name: "Destruction Derby Arenas"
   region: "PAL-M6"
+  gsHWFixes:
+    disableDepthSupport: 1 # Removes fog-like filter.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:2912]: see comment for other DDS entry

File: bin/resources/GameIndex.yaml (hunk 1 above)
Comment (RedDevilus) [reply bin/resources/GameIndex.yaml:2535]: Oh okay

File: bin/resources/GameIndex.yaml [hunk 3]
```diff
@@ -941,6 +953,10 @@ SCAJ-20165:
 SCAJ-20166:
   name: "Front Mission 5 - Scars of the War"
   region: "NTSC-Unk"
+  gameFixes:
+    - SoftwareRendererFMVHack # Flickering in FMV.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:957]: extra tab at end

File: bin/resources/GameIndex.yaml [hunk 4]
```diff
@@ -834,9 +841,14 @@ SCAJ-20141:
 SCAJ-20143:
   name: "Armored Core - Last Raven"
   region: "NTSC-Unk"
+  gsHWFixes:
+    preloadFrameData: 1 # Fixes black screen while in a mech.
+    roundSprite: 2 # Fixes HUD artifacts.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:846]: extra tab at end

File: bin/resources/GameIndex.yaml [hunk 5]
```diff
@@ -428,6 +428,11 @@ SCAJ-20056:
 SCAJ-20057:
   name: "Front Mission 4"
   region: "NTSC-Unk"
+  gameFixes:
+    - SoftwareRendererFMVHack # Flickering in FMV.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:432]: extra tab at end

File: bin/resources/GameIndex.yaml [hunk 6]
```diff
@@ -16071,6 +16121,9 @@ SLES-53819:
 SLES-53820:
   name: "Armored Core - Last Raven"
   region: "PAL-E"
+  gsHWFixes:
+    preloadFrameData: 1 # Fixes black screen while in a mech.
+    roundSprite: 2 # Fixes HUD artifacts.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:16126]: extra tab at end

File: bin/resources/GameIndex.yaml [hunk 7]
```diff
@@ -17543,6 +17598,9 @@ SLES-54466:
   compat: 5
   gameFixes:
     - VIFFIFOHack # Needed to load the main game properly.
+  gsHWFixes:
+    halfPixelOffset: 1 # Fixes bloom misalignment.
+    preloadFrameData: 1 # Fixes car reflections and brightness.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:17603]: extra tab at end

File: bin/resources/GameIndex.yaml [hunk 8]
```diff
@@ -24778,6 +24838,11 @@ SLPM-65442:
 SLPM-65443:
   name: "Front Mission 4"
   region: "NTSC-J"
+  gameFixes:
+    - SoftwareRendererFMVHack # Flickering in FMV.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:24842]: extra tab at end

File: bin/resources/GameIndex.yaml [hunk 9]
```diff
@@ -27189,6 +27258,10 @@ SLPM-66205:
   name: "Front Mission 5 - Scars of the War"
   region: "NTSC-J"
   compat: 5
+  gameFixes:
+    - SoftwareRendererFMVHack # Flickering in FMV.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:27262]: extra tab at end

File: bin/resources/GameIndex.yaml [hunk 10]
```diff
@@ -27909,9 +27982,18 @@ SLPM-66419:
 SLPM-66420:
   name: "Front Mission 4 [Ultimate Hits]"
   region: "NTSC-J"
+  gameFixes:
+    - SoftwareRendererFMVHack # Flickering in FMV.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:27986]: extra tab at end

File: bin/resources/GameIndex.yaml [hunk 11]
```diff
@@ -27909,9 +27982,18 @@ SLPM-66419:
...
+  gameFixes:
+    - SoftwareRendererFMVHack # Flickering in FMV.	
+  gsHWFixes:
+    preloadFrameData: 1 # Fixes mech shadows.
+    roundSprite: 2 # Fixes font artifacts.
 SLPM-66421:
   name: "Front Mission 5 - Scars of the War [Ultimate Hits]"
   region: "NTSC-J"
+  gameFixes:
+    - SoftwareRendererFMVHack # Flickering in FMV.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:27994]: extra tab at end

File: bin/resources/GameIndex.yaml [hunk 12]
```diff
@@ -32916,6 +32999,9 @@ SLPS-25461:
 SLPS-25462:
   name: "Armored Core - Last Raven"
   region: "NTSC-J"
+  gsHWFixes:
+    preloadFrameData: 1 # Fixes black screen while in a mech.
+    roundSprite: 2 # Fixes HUD artifacts.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:33004]: extra tab at end

File: bin/resources/GameIndex.yaml [hunk 13]
```diff
@@ -34877,6 +34965,9 @@ SLPS-73246:
 SLPS-73247:
   name: "Armored Core - Last Raven [PlayStation 2 The Best]"
   region: "NTSC-J"
+  gsHWFixes:
+    preloadFrameData: 1 # Fixes black screen while in a mech.
+    roundSprite: 2 # Fixes HUD artifacts.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:34970]: extra tab at end

File: bin/resources/GameIndex.yaml [hunk 14]
```diff
@@ -38700,6 +38800,11 @@ SLUS-20888:
   name: "Front Mission 4"
   region: "NTSC-U"
   compat: 5
+  gameFixes:
+    - SoftwareRendererFMVHack # Flickering in FMV.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:38804]: extra tab at end

File: bin/resources/GameIndex.yaml [hunk 15]
```diff
@@ -43996,6 +44108,11 @@ SLUS-29096:
 SLUS-29098:
   name: "Square Enix Sampler Disc Vol.2: Front Mission 4 Playable Demo"
   region: "NTSC-U"
+  gameFixes:
+    - SoftwareRendererFMVHack # Flickering in FMV.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:44112]: extra tab at end

**Pull request merged**